hle-survey/transformation/src/main/scala/org/broadinstitute/monster/dap/OwnerTransformations.scala

```diff
@@ -7,32 +7,41 @@ object OwnerTransformations {
   /** Parse all owner-related fields out of a raw RedCap record. */
   def mapOwner(rawRecord: RawRecord): HlesOwner = {
     val secondaryAddress = rawRecord.getBoolean("oc_address2_yn")
+    val raceValues = rawRecord.fields.get("od_race")
+
     HlesOwner(
       // FIXME: Once DAP figures out a name for a dedicated owner ID, use that.
       ownerId = rawRecord.id,
-      odAgeRangeYears = rawRecord.getOptional("od_age"),
-      odMaxEducation = rawRecord.getOptional("od_education"),
-      odMaxEducationOther = rawRecord.getOptional("od_education_other"),
-      odRace = rawRecord.getArray("od_race"),
-      odRaceOther = rawRecord.getOptional("od_race_other"),
+      odAgeRangeYears = rawRecord.getOptionalNumber("od_age"),
+      odMaxEducation = rawRecord.getOptionalNumber("od_education"),
+      odMaxEducationOtherDescription = rawRecord.getOptional("od_education_other"),
+      odRaceWhite = raceValues.map(_.contains("1")),
```

Comment:
@danxmoran I think you had suggested doing something like `Some(rawRecord.getArray("od_race").contains("1"))` to get these values but I'm concerned that doing so would set the value to `Some(false)` when it should be undefined, so I implemented it a bit differently. Let me know if you disagree or think there is a better way of doing it.

Reply:
I like it!